static/js/containers/pages/login/LoginPasswordPage.js

```diff
@@ -44,24 +44,24 @@ class LoginPasswordPage extends React.Component<Props> {
   async onSubmit({ password }, { setSubmitting, setErrors }) {
     const {
       loginPassword,
-      getCurrentUser,
-      history,
+      // getCurrentUser,
+      // history,
```

Comment:
Remove these lines and others below (and in `RegisterDetailsPage.js`) that have been commented out